internal/armada/server/submit.go

```diff
@@ -253,13 +268,30 @@ func (server *SubmitServer) DeleteQueue(ctx context.Context, request *api.QueueD
 func (server *SubmitServer) SubmitJobs(ctx context.Context, req *api.JobSubmitRequest) (*api.JobSubmitResponse, error) {
 	principal := authorization.GetPrincipal(ctx)
 
-	jobs, e := server.createJobs(req, principal.GetName(), principal.GetGroupNames())
+	jobs, responseItems, e := server.createJobs(req, principal.GetName(), principal.GetGroupNames())
 	if e != nil {
```

Comment:
Is there any limit to the size of the response we can generate here?  E.g. if the user submits a batch of 100 jobs and they all fail with some massive error are we sure we're not going to generate a response that exceeds  the max grpc response size? Likewise (although admittedly loess likely) could we end up generaling such a massive message that we would OOM.

If it were me I'd try and limit the size of the error message by saying:
- how may jobs failed validation (out of how may submitted)
- detailed error messages for the first n jobs that failed.  (maybe n=5 or something)

Reply:
@d80tb7  I'll replace that potentially-huge error with a  concise/distilled one, using that logic as you suggest.

Reply:
@d80tb7 An attempt to implement the logic described above is now in commit 894c265a - let me know what you think.